src/Bigcommerce/Api/Resources/RuleCondition.php

```diff
@@ -24,11 +24,11 @@ class RuleCondition extends Resource
 
 	public function create()
 	{
-		return Client::createResource('/products/' . $this->product_id . '/rules/' . $this->rule_id . '/conditions' , $this->getCreateFields());
+		return Client::createResource('/products/' . $this->product_id . '/rules/' . $this->fields->rule_id . '/conditions' , $this->getCreateFields());
```

Comment:
Took a look here--this one (and the one below) have to use `$this->product_id` because they have a property with that name.